.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         submodules: true
 
     - name: Use OCaml 4.14.0
-      uses: ocaml/setup-ocaml@v2
+      uses: AbstractMachinesLab/setup-ocaml@arm-support
```

Comment:
@ostera nit: can we gate this on arm64?

Reply:
So we keep the actions independent if some specific changes are needed in future.

Reply:
done 🆗